packages/wonder-blocks-tooltip/components/tooltip-anchor.test.js

```diff
@@ -0,0 +1,488 @@
+                // Act
+                const result = ref && ref.getAttribute("tabindex");
+
+                // Assert
+                expect(result).toBeNull();
+                done();
+            };
+
+            arrange(actAndAssert);
+        });
+
+        test("if we had added tabindex, removes it", (done) => {
+            // Arrange
+            const arrange = (actAndAssert) => {
+                const fakeTooltipPortalMounter = (((
+                    <View>This is the anchor</View>
+                ): any): React.Element<typeof TooltipPortalMounter>);
+
+                const TestFixture = (props: any) => (
+                    <View>
+                        <TooltipAnchor
+                            forceAnchorFocusivity={props.force}
+                            anchorRef={actAndAssert}
+                        >
+                            {(active) => fakeTooltipPortalMounter}
+                        </TooltipAnchor>
+                    </View>
+                );
+                const wrapper = mount(<TestFixture force={true} />);
+                wrapper.setProps({force: false});
+            };
+
+            const actAndAssert = (ref) => {
+                // Act
+                // Need to do a timeout so that the mount can return and the
+                // wrapper can change the force prop to false.
+                setTimeout(() => {
+                    const result = ref && ref.getAttribute("tabindex");
+
+                    // Assert
+                    expect(result).toBeNull();
+                    done();
+                }, 0);
+            };
+
+            arrange(actAndAssert);
+        });
+
+        test("if we had not added tabindex, leaves it", (done) => {
+            // Arrange
+            const arrange = (actAndAssert) => {
+                const fakeTooltipPortalMounter = (((
+                    <View tabIndex="-1">This is the anchor</View>
+                ): any): React.Element<typeof TooltipPortalMounter>);
+
+                const TestFixture = (props: any) => (
+                    <View>
+                        <TooltipAnchor
+                            forceAnchorFocusivity={props.force}
+                            anchorRef={actAndAssert}
+                        >
+                            {(active) => fakeTooltipPortalMounter}
+                        </TooltipAnchor>
+                    </View>
+                );
+                const wrapper = mount(<TestFixture force={true} />);
+                wrapper.setProps({force: false});
+            };
+
+            const actAndAssert = (ref) => {
+                // Act
+                // Need to do a timeout so that the mount can return and the
+                // wrapper can change the force prop to false.
+                setTimeout(() => {
+                    const result = ref && ref.getAttribute("tabindex");
+
+                    // Assert
+                    expect(result).not.toBeNull();
```

Comment:
I was misreading these tests before.  I thought that `forceAnchorFocusivity={false}` was resulting in `tabIndex` being removed, but that isn't the case.